src/app/src/components/Dashboard.jsx

```diff
@@ -84,7 +84,7 @@ function Dashboard({
                                             flag={CLAIM_A_FACILITY}
                                             alternative={DASHBOARD_TITLE}
                                         >
-                                            Dashboad / Facility Claim Details
+                                            Dashboard / Facility Claim Details
```

Comment:
This is a trivial change but I noticed it while checking on how I set up the Dashboard routing so I incorporated the fix into this commit.